src/continuum_opacity/opacity_H.jl

```diff
@@ -0,0 +1,237 @@
+              - 1.39568e-18 * λ5 + 2.78701e-23 * λ6)
+    αbf_H⁻ * 1e-18
+end
+
+
+"""
+    Hminus_bf(nHminus, ν, ρ, T, [ion_energy_H⁻])
+
+Compute the H⁻ bound-free opacity κ
+
+# Arguments
+- `nH_I_div_partition::Flt`: the total number density of H I divided by its partition function.
+- `ν::Flt`: frequency in Hz
+- `ρ::Flt`: mass density in g/cm³
+- `T::Flt`: temperature in K
+- `ion_energy_H⁻::Flt`: Specifies the ionization energy of the single state of H⁻ in eV. This is
+   roughly 0.7552 eV.
+
+# Notes
+This function assumes that n(H⁻) ≪ n(H I) + n(H II). The number density of n(H⁻) should not be
+pre-computed (instead it's computed internally by this function).
+
+This function is adapted from equation 8.12 from Grey (2005). This equation gives the absorption
+coefficient per H I atom (uncorrected by stimulated emission) is given by:
+    αff_H⁻ * 8.316e-10 * Pₑ * θ^2.5 * 10^(ion_energy_H⁻ * θ) * (U(H⁻,T)/ U(H I,T))
+where:
+- αff_H⁻ is the photo dissociation cross-section. This can estimated with equation 8.11.
+- θ = log10(e)/(k*T) or θ = 5040/T in units of eV⁻¹
+- U(H⁻,T) is the partition function of H⁻ at temperature T. This is always 1
+- U(H I,T) is the partition function of H I at temperature T. This is 2 at low to intermediate T.
+Eqn 8.12 of Grey (2005) implicitly assumes that (U(H⁻,T)/ U(H I,T)) is always equal to 0.5.
+
+This expression is simplicitly a rewritten form of: αff_H⁻ * n(H⁻)/n(H I) where n(H⁻)/n(H I) has
+been replaced with the expanded form of the saha equation, in which n₀ = n(H⁻) and n₁ = n(H I).
+
+Combining 8.18, and 8.19 of Gray (2005), indicate that the version opacity contribution of H⁻
+bound-free absorption (with stimulated emission correction) is given by:
+                     n(H⁻)                                  n(H I)          n(H I) + n(H II)
+   κ_ν = α_bf(H⁻) * ------  * (1 - exp(-h*ν/(k*T))) * ------------------ * -----------------
+                    n(H I)                             n(H I) + n(H II)            ρ
+This can be rewritten as: κ_ν = α_bf(H⁻) * n(H⁻) * (1 - exp(-h*ν/(k*T))) / ρ
+
+This function uses the polynomial provided in equation 8.11 of Gray (2005), that fits the tabulated
+data from Wishart (1979). While Gray (2005) claims that the polynomial fits the data with 0.2%
+precision for 2250 Å ≤ λ ≤ 15000 Å, in practice we find that it fits the data to better than 0.25%
+precision. Wishart (1979) expects the tabulated data to have better than 1% percent accuracy.
+"""
+function Hminus_bf(nH_I_div_partition::Flt, ne::Flt, ν::Flt, ρ::Flt, T::Flt,
+                   ion_energy_H⁻::Flt = 0.7552) where {Flt<:AbstractFloat}
+    αbf_H⁻ = _Hminus_bf_cross_section(ν) # does not include contributions from stimulated emission
+    stimulated_emission_correction = (1 - exp(-hplanck_cgs*ν/(kboltz_cgs*T)))
+    n_H⁻ = _ndens_Hminus(nH_I_div_partition, ne, T, _H⁻_ion_energy)
+    αbf_H⁻ * n_H⁻ * stimulated_emission_correction / ρ
+end
+
+
+
+"""
+    Hminus_ff(nHI_gs, ne, ν, ρ, T)
```

Comment:
The name of the first argument doesn't match the code.

Reply:
Got it. I'll fix it